llvm/lib/Target/AArch64/AArch64InstrFormats.td

```diff
@@ -13065,6 +13065,7 @@ class BaseCmpBranchRegister<RegisterClass regtype, bit sf, bits<3> cc,
       Sched<[WriteBr]> {
   let isBranch = 1;
   let isTerminator = 1;
+  let isCompare = 1;
```

Comment:
Do these need to be marked as compares? I believe we go through analyzeBranch. Same for the pseudos.

Reply:
Yep, thanks for catching that!